<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="next_btn">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>840</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>840</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="launch_btn">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>840</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Launch Game</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabs">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1101</width>
      <height>811</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="intro">
     <attribute name="title">
      <string>Introduction</string>
     </attribute>
     <widget class="QTextEdit" name="intro_textbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>1081</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="front_screen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1081</width>
        <height>591</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../images/all_urns_colour.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="disclaimer">
     <attribute name="title">
      <string>Disclaimer</string>
     </attribute>
     <widget class="QCheckBox" name="consent_checkbox">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>760</y>
        <width>571</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>I confirm my consent to the test and for my data to be stored and processed</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="disclaimer_textbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1081</width>
        <height>751</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="demo">
     <attribute name="title">
      <string>Demographics</string>
     </attribute>
     <widget class="QLineEdit" name="username_textbox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>31</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="age_spinbox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>67</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="edu_combobox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>146</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="gender_combobox">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>107</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>68</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>150</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Education Level</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="instr">
     <attribute name="title">
      <string>Test Instructions</string>
     </attribute>
     <widget class="QTextEdit" name="instr_textbox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1081</width>
        <height>781</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="debrief">
     <attribute name="title">
      <string>Debrief</string>
     </attribute>
     <widget class="QTextEdit" name="debrief_textbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1081</width>
        <height>781</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextEdit" name="error_textbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>812</y>
      <width>811</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_btn">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>840</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save and Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
